src/hnsw/build.c

```diff
@@ -314,28 +315,42 @@ static void ScanTable(HnswBuildState *buildstate)
 static void BuildIndex(
     Relation heap, Relation index, IndexInfo *indexInfo, HnswBuildState *buildstate, ForkNumber forkNum)
 {
+    InitBuildState(buildstate, heap, index, indexInfo);
```

Comment:
Note that per convention all local variable declarations should come before any statements;